tests/inject/dynamic/media-query.tests.ts

```diff
@@ -77,10 +77,30 @@ describe('Handle Media Queries', () => {
 
         createOrUpdateDynamicTheme(theme, null, false);
         await timeout(100);
-        
+
         expect(getComputedStyle(document.querySelector('h1')).backgroundColor).toBe('rgb(0, 102, 0)');
         expect(getComputedStyle(document.querySelector('h1 strong')).color).toBe('rgb(255, 174, 26)');
         expect(getComputedStyle(document.body).backgroundColor).toBe('rgb(0, 0, 0)');
     });
-    
+
+    it('should check for CSS @media', async () => {
```

Comment:
I remember you had a test with `<style media="...">`. I think it's worth adding it too along with testing that `<style media="print">` is ignored.